### PR TITLE
added acpid page from old wiki

### PR DESCRIPTION
**Pull request by ectlunya**
I added a brief section about busybox acpid which I use

### CODE REVIEW

File: software/acpid.txt [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+
++------------------------------------------------------------------------------+
+|                                                                              |
+|   #!/bin/sh                                                                  |
+|                                                                              |
+|   printf mem > /sys/power/state                                              |
+|                                                                              |
++------------------------------------------------------------------------------+
+
+The rest of this article uses the more robust acpid program from the acpid package in the community repository. Install it as usual:
```
Comment (dylanaraps) [inline software/acpid.txt:27]: Exceeds line length.

File: software/acpid.txt [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+|                                                                              |
++------------------------------------------------------------------------------+
+
+This will match any detected event and run the action file (see below).
+
+
+The handler script
+________________________________________________________________________________
+
+During an event, acpid will run a corresponding action. Since the events are passed literally as
```
Comment (dylanaraps) [inline software/acpid.txt:84]: Exceeds line length.

File: software/acpid.txt [hunk 3]
```diff
@@ -0,0 +1,123 @@
...
++------------------------------------------------------------------------------+
+|                                                                              |
+|   $ sv restart acpid                                                         |
+|                                                                              |
++------------------------------------------------------------------------------+
+
+References
+________________________________________________________________________________
+
+[0] More info at https://wiki.gentoo.org/wiki/ACPI#Kernel
```
Comment (dylanaraps) [inline software/acpid.txt:123]: Drop `More info at `.

File: software/acpid.txt [hunk 4]
```diff
@@ -0,0 +1,123 @@
...
+Make sure that ACPI-related kernel options are enabled. [0]
+
+
+Using busybox acpid
+________________________________________________________________________________
+
+Busybox provides a very limited acpid program. The only event that it works
+well for is closing the lid. For some users this is enough. To run
+commands when the lid is closed, create an executable script at
+/etc/acpi/LID/00000080. The example below will suspend your computer to RAM.
```
Comment (dylanaraps) [inline software/acpid.txt:17]: This paragraph can be re-wrapped. Lines can be longer in other words.

File: software/acpid.txt (hunk 1 above)
Comment (dylanaraps) [reply software/acpid.txt:27]: Also, `acpid` is available in the official repositories.

File: software/acpid.txt [hunk 5]
```diff
@@ -0,0 +1,123 @@
...
+|   printf mem > /sys/power/state                                              |
+|                                                                              |
++------------------------------------------------------------------------------+
+
+The rest of this article uses the more robust acpid program from the acpid package in the community repository. Install it as usual:
+
++------------------------------------------------------------------------------+
+|                                                                              |
+|   kiss build acpid                                                           |
+|   kiss install acpid                                                         |
```
Comment (dylanaraps) [inline software/acpid.txt:32]: See: k1ss.org/wiki/help/page-template for how command installation should be formatted.

File: software/acpid.txt [hunk 6]
```diff
@@ -0,0 +1,123 @@
+ACPID
+________________________________________________________________________________
+
```
Comment (dylanaraps) [inline software/acpid.txt:3]: A table of contents would benefit this article. :+1:

File: software/acpid.txt [hunk 7]
```diff
@@ -0,0 +1,123 @@
...
+
+Prerequisites
+________________________________________________________________________________
+
+Make sure that ACPI-related kernel options are enabled. [0]
```
Comment (dylanaraps) [inline software/acpid.txt:8]: It'd be great to include the kernel configuration information here. See https://k1ss.org/wiki/kernel/config for formatting.

File: software/acpid.txt [hunk 8]
```diff
@@ -0,0 +1,123 @@
...
+NOTE: Some 'Fn + *' keys might not appear as they are hardware-bound (e.g. WLAN
+      toggle). Just test them all and see what works. For most laptops, this
+      will also detect lid events (e.g. opening or closing).
+
+
+Read all events
+________________________________________________________________________________
+
+Create a file under /etc/acpi/events/ (just create these directories if they
+are missing) containing the following:
```
Comment (dylanaraps) [inline software/acpid.txt:69]:
```
Save the following lines to a file in /etc/acpi/events/. 
```

Is there a file name requirement also?

Comment (ectlunya) [reply software/acpid.txt:69]: I don't think so. The Arch wiki page doesn't mention anything about it. The event= line seems to determine everything

File: software/acpid.txt (hunk 4 above)
Comment (ectlunya) [reply software/acpid.txt]: I'm not sure I understand what you mean, but I left a few long lines in the latest commit

File: software/acpid.txt [hunk 9]
```diff
@@ -0,0 +1,155 @@
...
+|   $ sv restart acpid                                                     |
+|                                                                          |
++--------------------------------------------------------------------------+
+
+
+    [0.1] Kernel Configuration
+    ____________________________________________________________________________
+
+    For now, check the Gentoo Wiki for details. [0] The important information
+    will be added to this page soon.
```
Comment (dylanaraps) [inline software/acpid.txt:53]: This would be best as `[1.0]` (its own section).

Comment (dylanaraps) [reply software/acpid.txt:53]: Feel free to skip over the "optional acpi drivers" section of the Gentoo Wiki. Just include the mandatory config option and feel free to refer back to Gentoo for the optional stuff.

File: software/acpid.txt [hunk 10]
```diff
@@ -0,0 +1,155 @@
...
+
+
+[1.0] Busybox acpid
+________________________________________________________________________________
+
+Busybox provides a very limited acpid program, that only works well for laptop
+lid events. Everytime the lid is closed, acpid will execute /etc/acpid/LID/00000080.
```
Comment (dylanaraps) [inline software/acpid.txt:60]: Line exceeds 80 columns.

File: software/acpid.txt [hunk 11]
```diff
@@ -0,0 +1,155 @@
...
+|                                                                              |
+|   #!/bin/sh                                                                  |
+|                                                                              |
+|   printf mem > /sys/power/state                                              |
+|                                                                              |
++------------------------------------------------------------------------------+
+
+[2.0] acpid2
+________________________________________________________________________________
+
```
Comment (dylanaraps) [inline software/acpid.txt:74]: A description about acpid2 would be nice. :+1:

File: software/acpid.txt (hunk 10 above)
Comment (ectlunya) [reply software/acpid.txt:60]: Didn't you say above to leave rewrap this paragraph with longer lines?

**Comment by ectlunya** [comment]
Lines 109 and 112 both have long path names that go a little over 80 characters. With a line break the paragraph wrapping would be kind of weird so I just left them over for now.

File: software/acpid.txt (hunk 10 above)
Comment (dylanaraps) [reply software/acpid.txt]: I meant that there was space at the end of each line (within the 80 column limit) where the paragraph could be re-wrapped (making each line longer while within limits).

File: software/acpid.txt [hunk 12]
```diff
@@ -0,0 +1,162 @@
...
+recieved. Depending on your hardware and kernel configuration, these events
+include closing a laptop lid, connecting to an AC power adapter, pressing
+certain buttons and more.
+
+
+Index
+________________________________________________________________________________
+
+- Usage                                                                    [0.0]
+
```
Comment (dylanaraps) [inline software/acpid.txt:14]: This newline can be removed.

File: software/acpid.txt [hunk 13]
```diff
@@ -0,0 +1,162 @@
...
+- Kernel Setup                                                             [1.0]
+
```
Comment (dylanaraps) [inline software/acpid.txt:16]: This newline can be removed.

**Comment by ectlunya** [comment]
Its been a few days, any changes you're still waiting for?

File: software/acpid.txt (hunk 11 above)
Comment (dylanaraps) [reply software/acpid.txt:86]: More information about `acpid2`. :+1:

File: software/acpid.txt [hunk 14]
```diff
@@ -0,0 +1,169 @@
...
+    NOTE: Some 'Fn + *' keys might not appear as they are hardware-bound (e.g.
+          WLAN toggle). Just test them all and see what works. For most laptops,
+          this will also detect lid events (e.g. opening or closing).
+
+
+    [2.2] Handling Events
+    ____________________________________________________________________________
+
+    When ACPI events are recieved, acpid looks for a file in /etc/acpi/events
```
Comment (dylanaraps) [inline software/acpid.txt:120]: `recieved` -> `received`

File: software/acpid.txt [hunk 15]
```diff
@@ -0,0 +1,169 @@
...
+self-explanatory, but the following notable options can be disabled:
+
++------------------------------------------------------------------------------+
+|                                                                              |
+|   CONFIG_ACPI_PROCFS_POWER    This option is deprecated                      |
+|   CONFIG_ACPI_EC_DEBUGFS      Potentially interferes with reboot             |
+|   CONFIG_ACPI_TABLE_UPGRADE   KISS does not use an initrd by default         |
+|   CONFIG_ACPI_DEBUG           Adds 50k to kernel size                        |
+|   CONFIG_ACPI_PCI_SLOT        Usually unnecessary                            |
+|   CONFIG_ACPI_CUSTOM_METHOD   Potential security flaw                        |
```
Comment (dylanaraps) [inline software/acpid.txt:63]: Expand on this?

File: software/acpid.txt [hunk 16]
```diff
@@ -0,0 +1,169 @@
...
+|                                                                              |
++------------------------------------------------------------------------------+
+
+
+[2.0] Busybox acpid
+________________________________________________________________________________
+
+Busybox provides a very limited acpid program, that only works well for laptop
+lid events. Every time the lid is closed, acpid will execute
```
Comment (dylanaraps) [inline software/acpid.txt:72]: Is this claim correct? The source code seems to include more events: https://git.busybox.net/busybox/tree/util-linux/acpid.c#n61

File: software/acpid.txt (hunk 11 above)
Comment (ectlunya) [reply software/acpid.txt:86]: I'm fairly certain it's just a more expansive version. It handles more events and is configured differently. Not sure what else you're looking for

File: software/acpid.txt (hunk 16 above)
Comment (ectlunya) [reply software/acpid.txt:72]: The others don't work. Hence, "only works well for laptop lid."  The other macros that are given aren't actually used anywhere else in the program. Its possible to define new keys in /etc/acpi.map and /etc/acpid.conf with raw keycodes, but the last time I looked into it there was very little documentation and I couldn't get it to work. I can look into it again and add something later if I figure it out.

**Comment by dylanaraps** [comment]
Any update on this?  Still missing a wider description of acpid2.  :+1:

**Comment by ectlunya** [comment]
I rearranged the acpid2 section to match busybox without subsections. I think it's more compact and easier to read. I'm not sure if you saw the logs from a few days ago, but mcpcpc and I were talking about simplifying some of the software articles. It seems unnecessary to tell users how to use the package manager and enable services in every article. More importantly, it distracts from the main purpose of the articles. If you insist, I can add it back, but I think it would improve the general readability of the wiki if we simplified the template.

**Pull request merged**